authors/apps/articles/views.py

```diff
@@ -1,13 +1,28 @@
 from django.shortcuts import get_object_or_404
-
 from rest_framework.permissions import IsAuthenticatedOrReadOnly
 from rest_framework import status
 from rest_framework.response import Response
 from rest_framework import viewsets
+from rest_framework.generics import GenericAPIView
+from rest_framework.exceptions import ValidationError
+from django.db.models import Avg
 
-from .models import Article
-from .serializers import ArticleSerializer
+from .serializers import ArticleSerializer, RatingSerializer
 from .renderers import ArticleJsonRenderer
+from .models import Article, Rating
+
+
+def get_article(slug):
+
+    try:
+        article = Article.objects.get(slug=slug)
+
+    except Exception:
+        return Response(
+            {"error": "Article not found"},
+            status=status.HTTP_404_NOT_FOUND
+        )
+    return article
```

Comment:
This is better right after the attempt to get an article in line 18. Always complete a try block before moving to the Except.